deploy/openshift/quay-app.yaml

```diff
@@ -213,7 +213,6 @@ objects:
           - containerPort: 8443
           volumeMounts:
           - name: configvolume
-            readOnly: false
```

Comment:
Why is this changed needed?

Reply:
openshift removes this field, causing qontract-reconcile to think that it should apply again.
no harm in keeping it, just makes sense to me to not have fields with default values that are removed.